core/include/debug.h

```diff
@@ -40,8 +35,8 @@ extern "C" {
 /**
  * @def ENABLE_DEBUG
  * @brief   This macro can be defined as 0 or other on a file-based level.
- *          If ENABLE_DEBUG is 0 @ref DEBUG() and @ref DEBUGF() will generate
- *          no output if not they will generate output.
+ *          If ENABLE_DEBUG is 0 @ref DEBUG() will generate no output if not
+ *          they will generate output.
```

Comment:
How about:
"`DEBUG()` will generate output only if `ENABLE_DEBUG` is non-zero" (or `!= 0`)